### PR TITLE
General cleanup of the image and backup script

### PR DESCRIPTION
**Pull request by KashifSaadat**

### CODE REVIEW

**Review by marcinc — APPROVED** [review]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
-FROM alpine:3.6
-ENV ETCD_VER v3.0.17
+FROM alpine:3.8
+
+ENV ETCD_VER v3.3.9
```
Comment (marcinc) [inline Dockerfile:3]: do we run that version everywhere now? I presume it's backward compatible with all v3.x?

**Review by marcinc — APPROVED** [review]
Still LG.

File: Dockerfile (hunk 1 above)
Comment (KashifSaadat) [reply Dockerfile:3]: Yeah it's backwards compatible, was just for some of the env var changes.

**Pull request merged**